app/assets/stylesheets/application.scss

```
@@ -1 +1,19 @@
 @import 'bootstrap';
+
+//aboutus main container
+.wrapperDiv {
+    width:100vw;
+    height: 90vh;
+    display: flex;
+}
+
+.aboutus-img{
+    width:50%;
+    height:50%;
+    object-fit: cover;
+}
+
+.aboutus-card{
+    height: 38rem;
+    align-content: center;
+}
```

```suggestion
.aboutus {
  &-img{
    width:50%;
    height:50%;
    object-fit: cover;
  }
  
  &-card{
    height: 38rem;
    align-content: center;
  }
}
```
sass concatentation helps with organizing classes